### ISSUE TITLE
New filter added for Messages

### ISSUE THREAD

**Issue by xiangce** [RedHatInsights/insights-frontend-assets#15]
https://github.com/RedHatInsights/insights-plugins/pull/737
~~~
ERROR_MESSAGE = 'error Error on attach: Node not found'

add_filter(Specs.messages, ERROR_MESSAGE)
~~~

**Comment by PaulWay** [comment]
What do I have to do for this?

**Comment by xiangce** [comment]
Resolved in #37

**Comment by kylape** [comment]
Can you please merge the plugin PR?  We will automatically include the filter after it's merged.
